tests/foreman/api/test_convert2rhel.py

```diff
@@ -328,6 +328,8 @@ def test_convert2rhel_oracle_with_pre_conversion_template_check(
     assert host_content['facts']['conversions::success'] == 'true'
     convert2rhel_facts = json.loads(oracle.execute('cat /etc/rhsm/facts/convert2rhel.facts').stdout)
     assert convert2rhel_facts['conversions.env.CONVERT2RHEL_THROUGH_FOREMAN'] == '1'
+    # https://issues.redhat.com/browse/RHELC-1737
+    target_os_name = 'Oracle' if is_open('RHELC-1737') else 'Red Hat'
```

Comment:
Oracle tests are failing currently, but I think this would return as `Oracle Linux`, could you check and confirm?

Reply:
Oracle 8 is failing because of template issue. The assertion will pass once I fix the template for oracle 8 which is currently I am working on.